File: app/src/main/java/io/mgba/Fragments/Settings/UIFragment.java
package io.mgba.Fragments.Settings;

import android.os.Bundle;
import android.support.v7.preference.PreferenceFragmentCompat;

import io.mgba.R;

public class UIFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        addPreferencesFromResource(R.xml.ui_settings);
    }

}

File: app/src/main/java/io/mgba/ui/fragments/settings/UIFragment.kt
package io.mgba.ui.fragments.settings

import android.os.Bundle

import androidx.preference.PreferenceFragmentCompat
import io.mgba.R

class UIFragment : PreferenceFragmentCompat() {

    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        addPreferencesFromResource(R.xml.ui_settings)
    }

}
